test/testData.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
This kind of files are normally named `mocks` and since what it's exporting is just *data* not *behavior* it can be done through a JSON file (`mocks.json`)